### ISSUE TITLE
gluegun new error when installed globally with yarn

### ISSUE THREAD

**Issue by cmckni3** [infinitered/gluegun#371]
### Steps to reproduce

```sh
yarn global add gluegun@next
gluegun new my-project
```

### Installation message

```sh
success Installed "gluegun@2.0.0-beta.7" with binaries:
      - gluegun
```

### Stacktrace

```
TypeError: The specified path does not exist: ~/.config/yarn/global/node_modules/gluegun/bin/../tsconfig.json
    at resolveSync (~/.config/yarn/global/node_modules/tsconfig/src/tsconfig.ts:74:9)
    at Object.loadSync (~/.config/yarn/global/node_modules/tsconfig/src/tsconfig.ts:142:16)
    at readConfig (~/.config/yarn/global/node_modules/ts-node/src/index.ts:424:18)
    at load (~/.config/yarn/global/node_modules/ts-node/src/index.ts:192:20)
    at service (~/.config/yarn/global/node_modules/ts-node/src/index.ts:370:32)
    at Object.register (~/.config/yarn/global/node_modules/ts-node/src/index.ts:379:5)
    at Object.<anonymous> (~/.config/yarn/global/node_modules/gluegun/bin/gluegun:9:22)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)```
```

### Versions

    node: 8.9.4
    yarn: 1.5.1

**Comment by jamonholmgren** [comment]
Replicated ... will look into this soon! In the meantime, try beta.6.

**Comment by cmckni3** [comment]
Yarn doesn't recognize `2.0.0-beta.5` or `2.0.0-beta.6`. Strange.

**Comment by cmckni3** [comment]
`2.0.0-beta.4` works 👍

**Comment by jamonholmgren** [comment]
Still intending to look into this.

**Comment by jamonholmgren** [comment]
#379 might fix this. I'll cut a new beta release to test it out when it's merged. It appears to work fine in my limited testing locally.

**Comment by jamonholmgren** [comment]
This is still an issue. In the meantime, I recommend installing via NPM: `npm i -g gluegun`.

**Comment by jamonholmgren** [comment]
So, ironically, this issue was caused (indirectly -- long story) by using `yarn` to publish the packages. 🤦‍♂️ 

This issue is fixed as of `gluegun-2.0.0-beta.10`.
